Sources/OpenAPIAsyncHTTPClient/AsyncHTTPClientTransport.swift

```diff
@@ -68,15 +68,24 @@ public struct AsyncHTTPClientTransport: ClientTransport {
         /// The HTTP client used for performing HTTP calls.
         public var client: HTTPClient
 
+        /// The default shared HTTP client.
+        ///
+        /// This is a workaround for the lack of a shared client
+        /// in AsyncHTTPClient. Do not use this value directly, outside of
+        /// the `Configuration.init(client:timeout:)` initializer, as it will
+        /// likely be removed in the future.
+        private static let sharedClient: HTTPClient = .init()
+
         /// The default request timeout.
         public var timeout: TimeAmount
 
         /// Creates a new configuration with the specified client and timeout.
         /// - Parameters:
         ///   - client: The underlying client used to perform HTTP operations.
+        ///     Provide nil to use the shared internal client.
         ///   - timeout: The request timeout, defaults to 1 minute.
-        public init(client: HTTPClient = .init(), timeout: TimeAmount = .minutes(1)) {
-            self.client = client
+        public init(client: HTTPClient? = nil, timeout: TimeAmount = .minutes(1)) {
+            self.client = Self.sharedClient
```

Comment:
A bit harsh to ignore the users choice, isn't? ;) 
```suggestion
            self.client = client ?? Self.sharedClient
```

Reply:
🤦‍♂️ Mondays, amirite